src/components/Page/Page.tsx

```diff
@@ -32,7 +32,14 @@ export interface Props extends HeaderProps {
   forceRender?: boolean;
 }
 
-export type ComposedProps = Props & WithAppProviderProps;
+export interface DeprecatedProps {
+  /** Decreases the maximum layout width. Intended for single-column layouts
+   * @deprecated As of release 4.0, replaced by {@link https://polaris.shopify.com/components/structure/page#props-narrow-width}
```

Comment:
We don't currently use these tags in the style guide, but ids should be added to the props none the less

Reply:
I prefer this as well—using the correct tags and following standards. That way it is there if we want to support it in the future and/or change to tool we used to parse this, we have them.